netbout-web/src/main/java/com/netbout/rest/XeAlias.java

```diff
@@ -95,10 +100,14 @@ private static XeSource source(final Base base,
      */
     private static XeSource source(final Alias alias) throws IOException {
         final String email;
-        if (alias.email().contains("!")) {
-            email = alias.email().substring(0, alias.email().indexOf('!'));
+        final String newEmail;
+        if (alias.email().contains(EMAIL_SEPARATOR)) {
```

Comment:
@mkordas there is no need to do this check before split.
If there is no splitting symbol in the string you will get an array with single element containing the whole string.
Otherwise, the string will be split.
```java
final String newEmail;
final String[] emails = alias.email().split("!")
if (emails.length > 1) {
    email = emails[0];
    newEmail = emails[1];
} else {
    email = alias.email();
    newEmail = "";
}
```
This will simplify the logic a bit and make the ```EMAIL_SEPARATOR``` field unnecessary